tool/tctl/common/resource_command.go

```diff
@@ -3036,3 +3045,34 @@ func (rc *ResourceCommand) updateAccessMonitoringRule(ctx context.Context, clien
 	fmt.Printf("access monitoring rule %q has been updated\n", in.GetMetadata().GetName())
 	return nil
 }
+
+func (rc *ResourceCommand) createVnetConfig(ctx context.Context, client *authclient.Client, raw services.UnknownResource) error {
```

Comment:
should we add a "reset" func like we do for other singleton resources on `tctl rm`?
ex:
```go
	case types.KindClusterNetworkingConfig:
		if err = resetClusterNetworkingConfig(ctx, client); err != nil {
			return trace.Wrap(err)
		}
		fmt.Printf("cluster networking configuration has been reset to defaults\n")
```

Reply:
i don't think so, for the types where we do this, they can also be configured via config file, and always exist in the backend and get written during auth startup. for a `vnet_config` you can't configure it via config file and it's fine for it to not exist or be deleted